features/onbekend_waardes.feature

```diff
@@ -141,125 +146,150 @@ Rule: een veld van type Waardetabel wordt niet opgenomen wanneer de code de onbe
   | kinderen.geboorte.land                      | 0000            |
 
   Abstract Scenario: onbekend waarde voor <groep> <veld>
-    Gegeven het systeem heeft een persoon met de volgende gegevens
-    | naam                | waarde    |
-    | burgerservicenummer | 999992934 |
-    En de persoon heeft de volgende '<groep>' gegevens
+    Gegeven de persoon met burgerservicenummer '000000206' heeft de volgende gegevens
     | naam      | waarde   |
     | <element> | <waarde> |
     Als personen wordt gezocht met de volgende parameters
     | naam                | waarde                             |
     | type                | RaadpleegMetBurgerservicenummer    |
-    | burgerservicenummer | 999992934                          |
+    | burgerservicenummer | 000000206                          |
     | fields              | burgerservicenummer,<groep>.<veld> |
-    Dan heeft de response een persoon met alleen de volgende gegevens
+    Dan heeft de response een persoon met de volgende gegevens
     | naam                | waarde    |
-    | burgerservicenummer | 999992934 |
+    | burgerservicenummer | 000000206 |
+    En heeft de persoon een leeg 'geboorte' object
 
     Voorbeelden:
     | groep          | veld       | element                           | waarde |
     | geboorte       | plaats     | geboorteplaats (03.20)            | 0000   |
     | geboorte       | land       | geboorteland (03.30)              | 0000   |
+
+  Abstract Scenario: onbekend waarde voor <groep> <veld>
+    Gegeven de persoon met burgerservicenummer '000000139' heeft de volgende 'overlijden' gegevens
+    | naam      | waarde   |
+    | <element> | <waarde> |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                             |
+    | type                | RaadpleegMetBurgerservicenummer    |
+    | burgerservicenummer | 000000139                          |
+    | fields              | burgerservicenummer,<groep>.<veld> |
+    Dan heeft de response een persoon met de volgende gegevens
+    | naam                          | waarde    |
+    | burgerservicenummer           | 000000139 |
+    | overlijden.indicatieOverleden | true      |
+
+    Voorbeelden:
+    | groep          | veld       | element                           | waarde | 
     | overlijden     | plaats     | plaats overlijden (08.20)         | 0000   |
     | overlijden     | land       | land overlijden (08.30)           | 0000   |
 
   Abstract Scenario: onbekend waarde voor: <element>
```

Comment:
Waarom is dit een abstract scenario als er maar 1 regel in de voorbeelden tabel staat?